conf/telemetry/OPENUAS/openuas_magnetodebug_ardrone2.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
```

Comment:
Thx for looking at it, THe answer is configs and values in it will be changed to other values asap. there is still a lot of stuff to be merged, If would have done it before a pullreqest, clean up would be postponed and postponed again... Now I have a "clean" start to merge all changes lurking in various branches before the end of the year, then with all in, a new pull request to be expected.